ProjectName/src/components/ServiceBell/ServiceBell.tsx

```diff
@@ -46,9 +46,9 @@ const ServiceBell = () => {
         onPress={() => {
           dispatch(toggleActive());
         }}>
-        {messages.length !== 0 && (
+        {messages.length != 0 && (
```

Comment:
Wow. Interesting. Could you please check is this `eslint` changed `!==` to `!=`?

Reply:
I found when I used the previous rules (default), there is a warning that I must use !== instead of !=. When I applied the new rules (you shared wth me), the warning disappeared ;]